backend/MoleculeOSSite/appsettings.json

```diff
@@ -1,4 +1,10 @@
 {
+  "Authentication": {
+    "JwtKey": "PRIVATE_KEY",
+    "JwtExpireDays": 4,
+    "JwtIssuer": "http:/moleculeapi.com"
+  },

```

Comment:
Ustawienia autentyfikacji, a w szczególności JwtKey powinny być w .env. Takich rzeczy nie możesz pokazywać publicznie w repozytorium.

Swoją drogą, JwtKey jest za krótki. Gdy testowałem kod to miałem przez to błędy. Musi być dłuższy ten ciąg znaków.